### ISSUE TITLE
Lilac: Rapid Response cherry-picks

### ISSUE THREAD

**Issue by pdpinch** [mitodl/edx-platform#254]
Rapid Response includes an integration with the instructor dashboard. Since there's no API for injecting features into the instructor dashboard, we added it to our fork. 

cherry-pick and merge the following commits and confirm that they work on our Lilac fork:

https://github.com/mitodl/edx-platform/commit/a271585547cbe0927a3920a80a24700004d80971 (*edited)
https://github.com/mitodl/edx-platform/commit/cd92cf3f081bf5142a3c878599b69bb272071aa1

**Comment by pdpinch** [comment]
This PR should be opened against the mitx/lilac branch.

**Comment by arslanashraf7** [comment]
@pdpinch, Just to add information on this one:

- While setting up lilac I noticed that one of the devstack repos didn't yet cut out a lilac branch e.g. `https://github.com/edx/frontend-app-payment`.
- While trying to setup the devstack without https://github.com/edx/frontend-app-payment and I noticed that there were missing docker images e.g. `forum, discovery`
- So I started to set up master branch of edX & devstack to test these commits.
- Also I noticed that we will need to first merge https://github.com/mitodl/edx-platform/issues/255 because rapid_response cherry-picks do some changes in the files added by canvas commits which produce conflicts.

Let me know if you'd suggest otherwise.
